Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: docs
+
+docs:
+	rm -rf docs/plugins
+	python _docs/render.py
+	rm -rf ./napari
+	git clone --depth 1 --filter=blob:none --sparse -b update-plugin-docs https://github.com/tlambert03/napari.git
```

Comment:
```suggestion
        git clone --depth 1 --filter=blob:none --sparse -b update-plugin-docs https://github.com/napari/napari.git
```

Reply:
that won't work because napari doesn't have the update-plugin-docs branch.  this needs to be changed after the update-plugin-docs PR gets merged.